mcp_proxy/adapters/claude.py

```
@@ -0,0 +1,38 @@
+                    "args": [
+                        "--directory",
+                        config_path,
+                        "run",
+                        "context_mcp.py"
+                    ]
+                }
+            }
+        }
+            
+        with open(config_path, 'w') as f:
+            json.dump(config, f, indent=4)
```

Pylint (3.3.7)

[error] Possibly using variable 'config' before assignment

(E0606)